src/processor/LakeOfFHIR.ts

```diff
@@ -111,32 +116,48 @@ export class LakeOfFHIR implements utils.Fishable {
     }
   }
 
+  async prepareDefs() {
+    this.assignMissingIds();
+    this.removeDuplicateDefinitions();
+    await this.defs.initialize();
+    const lakeMap = new Map<string, any>();
+    this.docs.forEach(wildFHIR => {
+      lakeMap.set(wildFHIR.path, wildFHIR.content);
+    });
+    const lakePackage = new InMemoryVirtualPackage(
+      { name: 'wild-fhir', version: '1.0.0' },
+      lakeMap,
+      {
+        log: (level: string, message: string) => {
+          logMessage(level, message);
+        }
+      }
+    );
+    return this.defs.loadVirtualPackage(lakePackage);
+  }
+
   fishForFHIR(item: string, ...types: utils.Type[]) {
     // The simplest approach is just to re-use the FHIRDefinitions fisher.  But since this.docs can be modified by anyone at any time
     // the only safe way to do this is by rebuilding a FHIRDefinitions object each time we need it.  If this becomes a performance
     // concern, we can optimize it later -- but performance isn't a huge concern in GoFSH. Note also that this approach may need to be
     // updated if we ever need to support fishing for Instances.
-    const defs = new FHIRDefinitions();
-    this.docs.forEach(d => defs.add(d.content));
-    return defs.fishForFHIR(item, ...types);
+    return this.defs.fishForFHIR(item, ...types);
   }
```

Comment:
Is this comment still applicable since we're not re-building a FHIRDefinitions object each time anymore?